### PR TITLE
A big pile of exercises

### PR DESCRIPTION
**Pull request by tgecho**
First, I'm terribly sorry about dumping a pile of exercises in a single PR. Unfortunately, since adding an exercise requires modifying config.json and elm-package.json in a way that will cause parallel PRs to conflict any time one of them is merged.

If you have the slightest concern about one or more of the commits, just let me know and I'll yank them out of the branch so you can merge the rest. Otherwise I don't want to waste any reviewer time waiting for me to update parallel PRs so you can merge each one safely.

Thanks for your patience @parkerl !

### CODE REVIEW

**Comment by parkerl** [comment]
Holy exercises Batman! I will review ASAP!

File: exercises/difference-of-squares/DifferenceOfSquaresTests.elm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import ElmTest exposing (..)
+import DifferenceOfSquares exposing (squareOfSum, sumOfSquares, difference)
+
+
+tests : Test
+tests =
+  suite
+    "DifferenceOfSquares"
+    [ suite
+        "Square the sum of the numbers up to the given number"
```
Comment (parkerl) [inline exercises/difference-of-squares/DifferenceOfSquaresTests.elm]: test descriptions should be all lowercase?

File: exercises/pangram/PangramTests.elm [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+tests : Test
+tests =
+  suite
+    "Pangram"
+    [ test
+        "sentence empty"
+        (assertEqual
+          False
+          (Pangram.isPangram "")
```
Comment (parkerl) [inline exercises/pangram/PangramTests.elm]: Is there a reason to not expose `isPanagram` and use it unqualified as you have done in your other tests?

File: exercises/difference-of-squares/DifferenceOfSquaresTests.elm (hunk 1 above)
Comment (tgecho) [inline exercises/difference-of-squares/DifferenceOfSquaresTests.elm]: Ah... I hadn't been consciously doing that, but I guess you're right.

File: exercises/pangram/PangramTests.elm (hunk 2 above)
Comment (tgecho) [inline exercises/pangram/PangramTests.elm]: Nope :)

**Comment by tgecho** [comment]
@parkerl If you like, I can squash these "fix up" commits into their appropriate exercise specific commits when you're done skimming through.

File: exercises/triangle/TriangleTests.elm [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import ElmTest exposing (..)
+import Triangle exposing (triangleKind)
+
+
+tests : Test
+tests =
+  suite
+    "triangleKind"
+    [ test
+        "Equilateral triangles have equal sides"
```
Comment (parkerl) [inline exercises/triangle/TriangleTests.elm]: lowercase?

Comment (tgecho) [inline exercises/triangle/TriangleTests.elm]: Got it

**Comment by parkerl** [comment]
@tgecho you are too fast! These look great. I am going to make you a member of the team and when you are ready you can merge this. Great work.

**Comment by tgecho** [comment]
:tada: Sounds good! You just caught me at a good moment :)

Thanks!

**Comment by tgecho** [comment]
This should be good to go.

**Comment by parkerl** [comment]
Ok I have asked @kytrinyx to set you up...in the meantime I'm merging this.

**Pull request merged**